test/test_log_format.yml

```
@@ -0,0 +1,17 @@
+# GIVEN a webserver with VIRTUAL_HOST set to web1.tld
+web1:
+  image: web
+  expose:
+    - "81"
+  environment:
+    WEB_PORTS: 81
+    VIRTUAL_HOST: web1.tld
+
+# WHEN nginx-proxy runs with DEFAULT_HOST set to web1.tld
+sut:
```

Specify the container name to guarantee you can successfully look it up from Python:

```suggestion
sut:
  container_name: sut
```